src/pages/items/EditItem.vue

```diff
@@ -25,7 +25,11 @@ export default {
     ...mapActions('items', ['updateItem']),
     save() {
       if (this.$refs.formItem.isValidForm()) {
-        this.updateItem({ id: this.$route.params.id, updates: this.item })
+        this.updateItem({
+          id: this.$route.params.id,
+          list: this.$route.params.list,
+          updates: this.item
+        })
```

Comment:
Looking at the number of mutations triggered in vuex it might not need to copy the object before edit it.
I just need to edit it with the action on the moment it is updated by the user.

This line might not be needed 
`this.item = Object.assign({}, itemFromStore)`